_api-reference/index-apis/stats.md

```diff
@@ -773,46 +808,9 @@ By default, the returned statistics are aggregated in the `primaries` and `total
 ```
 </details>
 
-#### Example request: Comma-separated list of indexes
-
-```json
-GET /testindex1,testindex2/_stats
-```
-{% include copy-curl.html %}
-
-#### Example request: Wildcard expression
-
-```json
-GET /testindex*/_stats
-```
-{% include copy-curl.html %}
-
-#### Example request: Specific stats
-
-```json
-GET /testindex/_stats/refresh,flush
-```
-{% include copy-curl.html %}
-
-#### Example request: Expand wildcards
-
-```json
-GET /testindex*/_stats?expand_wildcards=open,hidden
-```
-{% include copy-curl.html %}
-
-#### Example request: Shard-level statistics
-
-```json
-GET /testindex/_stats?level=shards
-```
-{% include copy-curl.html %}
-
-## Concurrent segment search
-
-Starting in OpenSearch 2.10, [concurrent segment search]({{site.url}}{{site.baseurl}}/search-plugins/concurrent-segment-search/) allows each shard-level request to search segments in parallel during the query phase. If you [enable the experimental concurrent segment search feature flag]({{site.url}}{{site.baseurl}}/search-plugins/concurrent-segment-search#enabling-the-feature-flag), the Index Stats API response will contain several additional fields with statistics about slices (units of work executed by a thread). These fields will be provided whether or not the cluster and index settings for concurrent segment search are enabled. For more information about slices, see [Concurrent segment search]({{site.url}}{{site.baseurl}}/search-plugins/concurrent-segment-search#searching-segments-concurrently).
+## Response fields
 
-The following table provides information about the added response fields.
+The following table provides information about the response fields.
```

Comment:
only having these new response fields specific to concurrent search looks incomplete. Ideally, we should add details about all the fields in the table or just explicitly say these are concurrent search related fields in the response.

Reply:
@sohami @ticheng-aws Fully agreed. Could you provide all the info about other fields please so I can include it here?

Reply:
@kolchfa-aws it looks like all the index stats are already documented in the node stats page here: https://opensearch.org/docs/latest/api-reference/nodes-apis/nodes-stats/#indices

Reply:
@jed326 Thanks! So should I remove this section and add these to the nodes-stats table?

Reply:
I think you already added these to the node stats table in this PR. I think it's good to have the search stats separately on this page too though.

Reply:
Thanks. I'll add a link to the nodes stats API so users can reference descriptions for all fields.